arcor2_AREditor/Assets/BASE/Scripts/Parameter.cs

```diff
@@ -270,6 +273,24 @@ public class Parameter : IO.Swagger.Model.ActionParameter {
            
         }
 
+
+
+        public static IParameter InitializePositionParameter(ParameterMetadata actionParameterMetadata, VerticalLayoutGroup layoutGroupToBeDisabled, GameObject canvasRoot, OnChangeParameterHandlerDelegate onChangeParameterHandler, string value, string type, bool linkable, CanvasGroup windowToHideWhenRequestingObj, ActionPoint actionPoint) {
+
+            LinkableDropdownPositions dropdownParameter = GameObject.Instantiate(ActionsManager.Instance.LinkableParameterDropdownPositionsPrefab).GetComponent<LinkableDropdownPositions>();
+            string selectedValue = null;
+            if (value != null) {
+                selectedValue = Parameter.GetValue<string>(value);
+            }
+            dropdownParameter.Init(actionParameterMetadata, type, selectedValue, layoutGroupToBeDisabled, canvasRoot, onChangeParameterHandler, windowToHideWhenRequestingObj, actionPoint, linkable);
+            //DropdownParameterPutData(dropdownParameter.DropdownParameter, data, selectedValue, actionParameterMetadata.Name, onChangeParameterHandler, actionParameterMetadata.Type);
```

Comment:
Commented code could be removed. If it is there for some reason, I would add a comment explaining why it is commented out.

Reply:
done